app/assets/stylesheets/modules/success-error.css

```diff
@@ -0,0 +1,36 @@
+.success-error {
+  width: 100px;
+  display: block;
+  margin: 40px auto 0; }
+
+.path {
```

Comment:
@aellispierce and I plan on reworking this together tomorrow but this is currently written in sass.

We can either change the file extension to `scss` or rewrite this into regular CSS. Given that this application doesn't use sass, I think the better approach is to stay consistent with the current styling practices and write this as CSS instead.